aws/centralized-monitoring/README.md

```diff
@@ -0,0 +1,32 @@
+# centralized-monitoring
+
+This is simultaneously a Terraform module as well as a repository housing an
+equivalent CloudFormation template, `ternary-monitoring-cfn.json`. Uploading a
+CloudFormation template to the AWS console is an exercise left to the reader,
+but see below for an example invocation of this module in Terraform.
+
+Either approach will create a `TernaryCMPMonitoringAgent` role whose ARN you
+should copy and paste into the `Metrics Collection Role ARN` field inside
+Ternary's AWS cloud configuration.
+
+To obtain the values for `ternary_service_account_email` and
+`ternary_service_account_uid`, look at your Ternary Admin page.
+
+If you are using the [aws-centralized-monitoring] script and are directing
+metrics to flow into the account where you applied the [payer-account] role,
+then you do not need to apply this. Simply use the same role ARN generated from
+the `payer-account` module.
+
+## Example terraform usage
+
+```hcl
+module "ternary-centralized-monitoring" {
+    source = "https://github.com/TernaryInc/ternary-onb-permissions.git//aws/centralized-monitoring?ref=master"
```

Comment:
Did this work for you?  

I got it to work with
```
source = "github.com/TernaryInc/ternary-onb-permissions//aws/centralized-monitoring?ref=docz"
```

Reply:
It will be `master` once merged, right?

Reply:
yea.. was testing with this branch at the time but, yes it should be "master" once this is merged.